app/test_split_integer.py

```diff
@@ -2,20 +2,31 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (
+        sum(split_integer(32, 6)) == 32
+    ), "5 + 5 + 5 + 5 + 6 + 6 should be equal to 32"
```

Comment:
Suggesting to work on your exception messages. Here `5 + 5 + 5 + 5 + 6 + 6`doesn't relate to the assert statement at all.
You could do something like this:
`The sum of the parts should be equal to the original value`